vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue

```diff
@@ -306,6 +342,43 @@ export default {
         imageArr.push(String(link));
       };
 
+    },
+    
+    // Verify input fields of the form are filled
+    isInputValid: function() {
```

Comment:
Can you remind me to bring this up during standup? I've written code similar to this and I wonder if it's the best approach from a clean code perspective. 

(To clarify, I'm not saying there is anything particularly wrong with it, I'm just curious if there is a better approach)

Reply:
Sure! If there's a better way to do it i'm all for it!